.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
Do we still need to have the possibility of creating blank issues for the users in jx-docs repo?

Reply:
Only blank is what we should have imo. Anything more sounds excessive to me honestly.

Reply:
@MarckK What do you think? Let me know, so as to finalize this PR.

Reply:
We should have blank issues.